ASP.NET Core/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>ASP_NET_Core</title>
+    <link href="~/favicon.ico" rel="shortcut icon" type="image/x-icon" />
+
+    @* Uncomment to use the HtmlEditor control *@
+    @* <script src="https://unpkg.com/devextreme-quill/dist/dx-quill.min.js"></script> *@
```

Comment:
I think we need to add this script to the resulting bundle. Even more, I think it is better to create a couple of gulp tasks with different resulting resources so that it will be easy to run the application and change resources without writing additional code.

Reply:
Also, we can add information to the template Readme file, so that anyone can adjust settings according to their requirements.

Reply:
Added